TypeScript.YAML-tmLanguage

```diff
@@ -26,11 +26,18 @@ repository:
       '1': { name: variable.ts }
     end: (?=$|[;,])
     patterns:
+    - include: '#ternary-operator'
```

Comment:
If what you mean is why we need a 'ternary-operator' as a new addition, there are 2 reasons I thought of:
1) To make (? + expression + :) together
2) #Type-annotation consumes " : followed by something" before we can get to #expression. Even if we change the order in which they are present. It will include : followed by something as #type-annotation instead of skipping it to look for an expression.